rust/rsc/src/rsc/main.rs

```diff
@@ -64,7 +70,59 @@ fn make_app(state: Arc<sea_orm::DatabaseConnection>) -> Router {
         )
 }
 
-fn launch_eviction(state: Arc<sea_orm::DatabaseConnection>, tick_interval: u64, deadline: i64) {
+async fn create_standalone_db() -> Result<DatabaseConnection, sea_orm::DbErr> {
+    let db = Database::connect("sqlite::memory:").await?;
+    Migrator::up(&db, None).await?;
+    Ok(db)
+}
+
+async fn create_remote_db(config: &config::RSCConfig) -> Result<DatabaseConnection, Box<dyn std::error::Error>> {
+    let connection = Database::connect(&config.database_url).await?;
+    let pending_migrations = Migrator::get_pending_migrations(&connection).await?;
+    if pending_migrations.len() != 0 {
+        let err = Error::new(
+            ErrorKind::Other,
+            format!(
+                "This rsc version expects {:?} additional migrations to be applied",
+                pending_migrations.len()
+            ),
+        );
+        tracing::error! {%err, "unperformed migrations, please apply these migrations before starting rsc"};
+        Err(err)?
+    }
+
+    Ok(connection)
+}
+
+async fn create_insecure_api_key(db: &DatabaseConnection) -> Result<String, Box<dyn std::error::Error>> {
+    let key = data_encoding::BASE64.encode(b"Insecure Key");
```

Comment:
Why not just make the key "insecurekey" instead? That way if you're writing a curl command you know what to type every time without memorizing some magic string

Reply:
oh that's just clearly better. Done